scripts/sync-down-from-production.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env sh
```

Comment:
Can I get an early review on this file?

Reply:
overall seems like a good, more simplified direction to me - thanks for this! appreciate the explicit pg role for accessing the `REMOTE_DB` and containerized approach to script dependencies here.

main open questions for me: 
- can we wrap psql commands starting from truncate in a transaction
- double check it won't fail if you have a local hasura migration on one of the synced tables